workflows/virology/influenza-isolates-consensus-and-subtyping/CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+## [0.2] - 2025-05-02
+
+### Changed
+
+Aaron Kolbecher, in the course of his Master thesis, discovered several
+edge-cases in which the previous version of the workflow would fail. The
+corresponding fixes listed below make the workflow much more robust and ready
+for use in high-throughput scenarios.
+
+- Added processing logic for handling diverse VAPOR failures.
+
+  Note: As a limitation of the new logic elements in the "References per segment"
+  collection can no longer contain colons (`:`) in their identifiers.
+
+- Improved subtyping report to handle cases of undetermined HA and/or NA subtypes.
+
+  The subtyping report will now contain H? and/or N? for unresolved subtypes.
+
+- Added logic for skipping multiple-sequence alignment of a genome segment if a
+  consensus sequence for that segment could be generated for only a single
+  sample (instead of causing a MAFFT error).
+
+- Added logic for skipping phylogenetic analysis with IQ-Tree for segments for
+  which fewer than three samples yielded a consensus sequence (instead of
+  causing an IQ-Tree error).
+
+  Note: IQ-Tree can still fail if a consensus sequence consists of Ns only.
+
+- No longer assume eight genome segments, but autodetect the number of segments
+  from the reference collection.
+
+  This makes it simpler to adapt the workflow to influenza species with seven
+  instead of eight segments, and makes it usable with partial reference
+  collections, for example if the purpose is to generate consensus sequences
+  only for certain segments of interest.
+
+- Added Aaron as a co-author of the workflow.
+
+- Changed license to AGPL-3.0-or-later.
```

Comment:
Can all authors please confirm they are ok with the license change?

Reply:
@wm75 @mvdbeek I'm OK with the change of license.

Reply:
@SaimMomin12 and @kciy confirmed, and me and Aaron have introduced the change in the new version. So that's all authors :-)
Thanks everybody!